wiki/Reporting_bad_behaviour/en.md

```diff
@@ -8,8 +8,9 @@ Accidental reports are ignored. Users who deliberately send invalid reports can
 
 | Nature of infringement | Point of contact |
 | :-- | :-- |
-| Offensive private messages | **None**: [ignore the user](/wiki/Client/Interface/Chat_console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Client/Options). |
-| Infringements related to modding and/or mapping | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or members of the [GMT](/wiki/People/The_Team/Global_Moderation_Team) focusing on mapping/modding community moderation, via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
+| Scamming/Phishing in private messages | Send a regular report |
+| Other offensive/inappropriate private messages | **None**: [ignore the user](/wiki/Client/Interface/Chat_console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Client/Options) or block them on the website with the button located on their user profile. |
+| Inappropriate conduct from a [BN](/wiki/en/People/The_Team/Beatmap_Nominators) | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
 | Inappropriate conduct from a [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member | [Account support team](/wiki/People/The_Team/Account_support_team) at [support@ppy.sh](mailto:support@ppy.sh) |
```

Comment:
```suggestion
| Inappropriate conduct from an [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member | [Account support team](/wiki/People/The_Team/Account_support_team) at [support@ppy.sh](mailto:support@ppy.sh) |
```

Reply:
I lean towards "a NAT", but only because "a NAT" itself is a common simplification of "a member of the NAT", same with "a GMT".

if we consider it only as an abbreviation, it should be "an NAT *(an, ay, tee)* member"

perhaps reword it, for both simplicity and less ambiguity, to `a member of the NAT/GMT`?

Reply:
```suggestion
| Inappropriate conduct from a member of the [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) | [Account support team](/wiki/People/The_Team/Account_support_team) at [support@ppy.sh](mailto:support@ppy.sh) |
```

Reply:
> ... It's not like omitted words influence the grammar here

I know, I have read the sentence until the end. this was used as an explanation for my preferences and why I would go against the current and read/write it not as an initialism (similarly to how we read "lol" instead of "al, oh, al", sadly I don't know an English term for that)